config/v1/types_infrastructure.go

```diff
@@ -308,7 +308,7 @@ type PlatformSpec struct {
 	// +optional
 	Nutanix *NutanixPlatformSpec `json:"nutanix,omitempty"`
 
-	// ExternalPlatformType represents generic infrastructure provider.
+	// External contains settings specific to the External platform.
```

Comment:
The previous comment seems to be incorrectly copy-pasted from https://github.com/openshift/api/pull/1574/files#diff-07b264a49084976b670fb699badaca1795027d6ea732a99226a5388104f6174fR213